app/controllers/meta_controller.rb

```diff
@@ -0,0 +1,23 @@
+class MetaController < ApplicationController
```

Comment:
You'll notice there's no `browserconfig` or `manifest` actions here even though they're mentioned in `routes.rb`. Turns out Rails will render views without controller actions. 🙃